schemas/context/placecontext.example.1.json

```diff
@@ -8,5 +8,19 @@
     "xdm:postalCode": "141-0032",
     "schema:latitude": 35.6185,
     "schema:longitude": 139.73237
+  },
+  "xdm:poiInteraction": {
+    "https://ns.adobe.com/xdm/data/metrics/placecontext/poi-exits": {
```

Comment:
is POI appropriate?

Reply:
While I dislike some of the evolutionary problems with "Point of Interest" POI as a term. It is a generally used and understood definition at this point that has a common meaning as we use it here.